packages/composer/amazeelabs/silverback_gatsby/tests/src/Kernel/EditorBlocksTest.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace Drupal\Tests\silverback_gatsby\Kernel;
+
+use Drupal\file\Entity\File;
+use Drupal\media\Entity\Media;
+use Drupal\node\Entity\Node;
+use Drupal\silverback_gutenberg\BlockSerializer;
+use Drupal\Tests\media\Traits\MediaTypeCreationTrait;
+use Drupal\Tests\TestFileCreationTrait;
+
+class EditorBlocksTest extends EntityFeedTestBase {
+  public static $modules = ['file', 'media', 'image', 'path_alias', 'silverback_gutenberg'];
+  use MediaTypeCreationTrait;
+  use TestFileCreationTrait;
+
+  protected function setUp(): void {
+    parent::setUp();
+    $this->installEntitySchema('media');
+    $this->installEntitySchema('file');
+    $this->installSchema('file', ['file_usage']);
+    $this->installConfig('media');
+    $this->createMediaType('image', ['id' => 'image']);
+    $this->container->get('content_translation.manager')->setEnabled(
+      'media',
+      'image',
+      TRUE
+    );
+  }
+
+  function testEditorBlockResolution() {
+
+    File::create([
+      'uri' => $this->getTestFiles('image')[0]->uri,
+    ])->save();
+    $media = Media::create([
+      'bundle' => 'image',
+      'name' => 'Screaming hairy armadillo',
+      'field_media_image' => [
+        [
+          'target_id' => 1,
+          'alt' => 'Screaming hairy armadillo',
+          'title' => 'Screaming hairy armadillo',
+        ],
+      ],
+    ]);
+    $media->addTranslation('de', [
+      'bundle' => 'image',
+      'name' => 'Screaming hairy armadillo',
+      'field_media_image' => [
+        [
+          'target_id' => 1,
+          'alt' => 'Screaming hairy armadillo DE',
```

Comment:
`Hairy armadillo screaming in German`?

Reply:
😆 copied that one from some drupal core unit test